### PR TITLE
DOCSP-27455 added customDns examples

### PR DESCRIPTION
**Pull request by kyuan-mongodb**
## Proposed changes

Added examples for the customDns aws commands.

_Jira ticket:_ DOCSP-27455

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/cli/atlas/customdns/aws/describe.go [hunk 1]
```diff
@@ -57,9 +57,11 @@ func DescribeBuilder() *cobra.Command {
 	opts := &DescribeOpts{}
 	cmd := &cobra.Command{
 		Use:     "describe",
-		Short:   "Describe the custom DNS configuration of an Atlas project’s cluster deployed to AWS.",
+		Short:   "Describe the custom DNS configuration of an Atlas project's cluster deployed to AWS.",
```
Comment (corryroot) [inline internal/cli/atlas/customdns/aws/describe.go:60]:
```suggestion
		Short:   "Describe the custom DNS configuration of an Atlas cluster deployed to AWS in the specified project.",
```

File: internal/cli/atlas/customdns/aws/disable.go [hunk 2]
```diff
@@ -55,8 +55,10 @@ func DisableBuilder() *cobra.Command {
 	opts := &DisableOpts{}
 	cmd := &cobra.Command{
 		Use:   "disable",
-		Short: "Disable the custom DNS configuration of an Atlas project’s cluster deployed to AWS.",
+		Short: "Disable the custom DNS configuration of an Atlas project's cluster deployed to AWS.",
```
Comment (corryroot) [inline internal/cli/atlas/customdns/aws/disable.go:58]:
```suggestion
		Short: "Disable the custom DNS configuration of an Atlas cluster deployed to AWS in the specified project.",
```

File: internal/cli/atlas/customdns/aws/enable.go [hunk 3]
```diff
@@ -55,8 +55,10 @@ func EnableBuilder() *cobra.Command {
 	opts := &EnableOpts{}
 	cmd := &cobra.Command{
 		Use:   "enable",
-		Short: "Enable the custom DNS configuration of an Atlas project’s cluster deployed to AWS.",
+		Short: "Enable the custom DNS configuration of an Atlas project's cluster deployed to AWS.",
```
Comment (corryroot) [inline internal/cli/atlas/customdns/aws/enable.go:58]:
```suggestion
		Short: "Enable the custom DNS configuration of an Atlas cluster deployed to AWS in the specified project.",
```

**Review by corryroot — DISMISSED** [review]
"Atlas project's cluster" reads awkwardly. I made some suggestions for clarity. Otherwise, LGTM.

File: docs/atlascli/command/atlas-customDns-aws-describe.txt [hunk 4]
```diff
@@ -77,3 +77,10 @@ If the command succeeds, the CLI returns output similar to the following sample.
    <Enabled>
    
 
+Examples
+--------
+
+.. code-block::
+
+   # Return the details for the custom DNS configuration deployed to AWS in the project with ID 618d48e05277a606ed2496fe:		
+   atlas customDns aws describe  --projectId 618d48e05277a606ed2496fe
```
Comment (tibulca) [inline docs/atlascli/command/atlas-customDns-aws-describe.txt:86]: [optional] There are 2 spaces between the command and  --projectId, is it intentional?

**Review by tibulca — DISMISSED** [review]
LGTM, small question

File: docs/atlascli/command/atlas-customDns-aws-describe.txt (hunk 4 above)
Comment (kyuan-mongodb) [reply docs/atlascli/command/atlas-customDns-aws-describe.txt]: Ah yes, sorry about that. I just fixed it :) @tibulca, would you mind LGTM-ing again?

**Review by tibulca — APPROVED** [review]

**Pull request merged**